src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/search.js.template

```diff
@@ -23,21 +23,21 @@
  * questions.
  */
 
-var noResult = {l: "No results found"};
-var loading = {l: "Loading search index..."};
-var catModules = "Modules";
-var catPackages = "Packages";
-var catTypes = "Types";
-var catMembers = "Members";
-var catSearchTags = "SearchTags";
+var noResult = {l: "##REPLACE:doclet.search.no_results##"};
+var loading = {l: "##REPLACE:doclet.search.loading##"};
+var catModules = "##REPLACE:doclet.search.modules##";
+var catPackages = "##REPLACE:doclet.search.packages##";
+var catTypes = "##REPLACE:doclet.search.types##";
+var catMembers = "##REPLACE:doclet.search.members##";
+var catSearchTags = "##REPLACE:doclet.search.search_tags##";
 var highlight = "<span class=\"result-highlight\">$&</span>";
 var searchPattern = "";
 var fallbackPattern = "";
 var RANKING_THRESHOLD = 2;
 var NO_MATCH = 0xffff;
 var MIN_RESULTS = 3;
 var MAX_RESULTS = 500;
-var UNNAMED = "<Unnamed>";
+var UNNAMED = "##REPLACE:doclet.search.unnamed##";
```

Comment:
`<Unnamed>` is not a string that is ever shown to the user, it is what is used by javadoc to denote the default package (see `toolkit.util.DocletConstants.DEFAULT_PACKAGE_NAME`). This variable shouldn't be localized as that would break behaviour for code in the default package (and show the localized string as package name, instead of no package name).

Reply:
Noted, thanks